bids/reports/parsing.py

```diff
@@ -469,7 +469,7 @@ def parse_niftis(layout, niftis, subj, config, **kwargs):
                 description_list.append(anat_info(suffix, metadata, img,
                                                   config))
             elif nifti_struct.entities['datatype'] == 'dwi':
-                bval_file = nii_file.replace('.nii.gz', '.bval')
+                bval_file = nii_file.replace('.nii', '.bval')
```

Comment:
This doesn't look right. Perhaps
```suggestion
                bval_file = nii_file.replace('.nii', '.bval').replace('.nii.gz', '.bval')
```

Reply:
I think you'd need to switch `.nii` and `.nii.gz` or it would end up with `.bval.gz` after the first replace.

Reply:
Ah yes. My bad.